manifests/v1beta1/installs/katib-external-db/kustomization.yaml

```diff
@@ -15,6 +15,8 @@ resources:
   - ../../components/ui/
   # Katib webhooks.
   - ../../components/webhook/
+  # Secret for webhooks certs.
+  - ./katib-webhook-cert.yaml
```

Comment:
I think, we can simplify this as follows in the Kustomization file:
```
secretGenerator:
  - name: katib-webhook-cert
    options:
      disableNameSuffixHash: true
```

WDYT @tenzen-y ?

Reply:
It's a good point! Thanks!

Reply:
Done.